pkgs/misc/emulators/ppsspp/default.nix

```diff
@@ -1,18 +1,22 @@
 { stdenv, fetchgit, zlib, libpng, qt4, pkgconfig
-, withGamepads ? true, SDL # SDL is used for gamepad functionality
+, withGamepads ? true, SDL ? null # SDL is used for gamepad functionality
```

Comment:
If `SDL` is `null`, then `withGamepads` have to be `false` by default, or I'm missing something?